lib/sqlalchemy/ext/asyncio/session.py

```diff
@@ -979,6 +979,12 @@ def begin_nested(self) -> AsyncSessionTransaction:
         For a general description of ORM begin nested, see
         :meth:`_orm.Session.begin_nested`.
 
+        .. seealso::
+
+            :ref:`aiosqlite_serializable` - special workarounds required
+            with the SQLite driver in order for SAVEPOINT to work
```

Comment:
**mike bayer** (zzzeek) wrote:

"with the SQLite asyncio driver"

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/sqlalchemy/+/4907